packages/node/package.json

```diff
@@ -10,7 +10,7 @@
   "scripts": {
     "test": "npm run test:npm && npm run test:yarn1",
     "test:prepare": "node ./test/prepare.js",
-    "test:npm": "cross-env LAVAMOAT_PM=npm@latest npm run test:prepare && ava",
```

Comment:
is the reason for removing `LAVAMOAT_PM=npm@latest` to fix what you mentioned earlier

> - This also fixes a problem where `npm@10.x` would complain if run under Node.js v16.x.

Reply:
and curious why `cross-env` isn't needed anymore for npm, but still needed for yarn
(and noticed the repo's archived/read-only, if that's one of the reasons)

Reply:
It's because it's easier to just update the default value in `prepare.js`.  `npm@latest` is not necessarily what we want to be using (since npm v10.x is incompatible with Node.js v16.x)

`cross-env` is basically "complete" afaict.  I'm not particularly concerned about it, but another solution would be to use `.env` files.  I'd suggest doing that exact thing in a year or two--once Node.js v20.x becomes the minimum supported version--since Node.js v20.x has native support for `.env` files.